rruleset.go

```diff
@@ -13,6 +15,28 @@ type Set struct {
 	exdate []time.Time
 }
 
+func (set *Set) String() string {
+	res := set.Recurrences()
+	return strings.Join(res, "\n")
+}
+
+func (set *Set) Recurrences() []string {
+	res := []string{}
+	for _, o := range set.rrule {
+		res = append(res, fmt.Sprintf("RRULE:%v", o.String()))
+	}
+	for _, o := range set.rdate {
+		res = append(res, fmt.Sprintf("RDATE:%v", o.Format(strformat)))
```

Comment:
'Z' in "20060102T150405Z" means UTC. So it should be `o.UTC().Format(strformat)`. Please use function `timeToStr`:
`res = append(res, fmt.Sprintf("RDATE:%s", timeToStr(o)))`